packages/amplify-util-mock/src/func/index.ts

```diff
@@ -58,7 +58,7 @@ export async function start(context: $TSContext): Promise<void> {
       printer.info(typeof result === 'undefined' ? '' : stringResult);
     } catch (err) {
       printer.error(`${resourceName} failed with the following error:`);
-      printer.info(err);
+      printer.info(err.message);
```

Comment:
Just to be safe to ensure no error is thrown in the catch block, please add an optional chaining operator:

```suggestion
      printer.info(err?.message);
```